### PR TITLE
Update `make react-native` usage

### PR DESCRIPTION
**Pull request by PombeirP**
Add required changes for https://github.com/status-im/status-react/pull/8149

### CODE REVIEW

File: source/build_status/desktop.md [hunk 1]
```diff
@@ -15,12 +15,14 @@ cd status-react
...
 ```
 
-For a Windows build cross-compiled from Linux, replace `make release-desktop` with `make release-windows-desktop`.
+Note: for a Windows build cross-compiled from Linux, replace `make release-desktop` with `make release-windows-desktop`.
 
 # Development environment setup
 
 ## To install react-native-cli with desktop commands support
 
+Comment out the `"react-native-cli-2.0.1" = nodeEnv.buildNodePackage` block in https://github.com/status-im/status-react/blob/develop/nix/global-node-packages/output/node-packages.nix and run:
```
Comment (jakubgs) [inline source/build_status/desktop.md]: This is pretty awful. Can't we make this easier on the user?

Comment (PombeirP) [reply source/build_status/desktop.md]: I agree, but my thought was that this is a pretty advanced scenario and probably not worth rethinking how we import this node package in order to make it flexible enough to switch among to distributions.

Comment (jakubgs) [reply source/build_status/desktop.md]: fair enough

**Comment by PombeirP** [comment]
@jakubgs can you please merge this PR? The corresponding PR in status-react has already been merged.

**Comment by jakubgs** [comment]
Deployments:
* https://ci.status.im/job/misc/job/dev.status.im/
* https://ci.status.im/job/misc/job/status.im/434/

**Pull request merged**